src/main/java/org/pac4j/vertx/handler/impl/CallbackHandler.java

```diff
@@ -51,75 +43,37 @@
     private final HttpActionAdapter httpActionHandler = new DefaultHttpActionAdapter();
     private final Vertx vertx;
     private final Config config;
+    private final boolean multiProfile;
 
     protected String defaultUrl = Pac4jConstants.DEFAULT_URL_VALUE;
```

Comment:
The `defaultUrl` has not value by default, this is handled by the `CallbackLogic`. How this parameter is set?

Why not getting rid of the `authHandlerOptions` and directly set the parameters (via constructors or setters).